### PR TITLE
[Feature/APP-430] Promotional cards: event

### PR DESCRIPTION
**Pull request by jdandrade**
THIS PR DEPENDS ON https://github.com/Aptoide/aptoide-client-v8/pull/1459

**What does this PR do?**

   This PR is the implementation of the promotional card: EVENT

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] BundlesResponseMapper.java

**How should this be manually tested?**

  Ask me for charles rewrite (for promotional card). Test views affected by the assets changes.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-430](https://aptoide.atlassian.net/browse/APP-430)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

### CODE REVIEW

**Review by phillipsgoncalves — CHANGES_REQUESTED** [review]
Also, noticed that the Skeletons of the textivews have the wrong color on the dark theme

**Comment by phillipsgoncalves** [comment]
Putting the comment here, so it does not get missed along with the review ones: 
"Also, noticed that the Skeletons of the textivews have the wrong color on the dark theme - they have the light one"

File: app/src/main/res/layout/card_event.xml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        />
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/card_title_label"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="8dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        app:cardBackgroundColor="#3c3c3c"
```
[the remark this answers is not in the document]
Comment (jdandrade) [reply app/src/main/res/layout/card_event.xml:89]: In the future we shall review this.

File: app/src/main/res/layout/card_event.xml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    <TextView
+        android:id="@+id/event_ongoing"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginTop="20dp"
+        android:text="@string/promotional_on_going_event"
+        android:textColor="@color/green"
```
[the remark this answers is not in the document]
Comment (jdandrade) [reply app/src/main/res/layout/card_event.xml:75]: Done

File: app/src/main/res/layout/card_event.xml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+          android:layout_height="wrap_content"
+          android:layout_gravity="center"
+          android:paddingBottom="5dp"
+          android:paddingEnd="10dp"
+          android:paddingLeft="10dp"
+          android:paddingRight="10dp"
+          android:paddingStart="10dp"
+          android:paddingTop="5dp"
+          android:textAllCaps="true"
+          android:textColor="@color/white"
```
[the remark this answers is not in the document]
Comment (jdandrade) [reply app/src/main/res/layout/card_event.xml:107]: Done

**Pull request merged**